### PR TITLE
Add Pivotal London event

### PR DESCRIPTION
**Pull request by deniseyu**

### CODE REVIEW

File: _data/events/London-Pivotal.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "title": "GDCR 17 @ Pivotal London",
+  "url": "https://www.eventbrite.com/e/global-day-of-code-retreat-pivotal-tickets-37119002937",
+  "moderators": [
+    "@deniseyu21",
+		"jamesjoshuahill",
+ 		"chewymeister"
+  ],
```
Comment (rradczewski) [inline _data/events/London-Pivotal.json]: Can you please add `@` to the handles so we can recognize them as twitter handles?

File: _data/events/London-Pivotal.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  "location": {
+    "city": "London",
+    "country": "UK",
```
Comment (rradczewski) [inline _data/events/London-Pivotal.json]: I guess it's going to happen at Pivotal, right? If you add the `location.coordinates` properties here the event will show up on the map as well :)

**Comment by deniseyu** [comment]
Made the changes you requested @rradczewski! Thanks :-)

**Comment by rradczewski** [comment]
Looks perfect, thanks and welcome to GDCR17! 👍

**Pull request merged**